spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigUtils.java

```diff
@@ -390,7 +399,7 @@ public static final class Prefix {
 		/**
 		 * prefix is known at the callsite.
 		 */
-		public static Prefix KNOWN;
+		public static Prefix KNOWN = new Prefix(() -> "", "KNOWN");
```

Comment:
this is a minor bug fix, that someone was not found until now, not really related to the fix itself